tests/unit/r2i/tflite/prediction.cc

```diff
@@ -92,7 +92,7 @@ TEST (TfLitePrediction, PredictionNonExistentIndex) {
   error = prediction.SetTensorValues(tensordata, INPUTS);
   LONGS_EQUAL (r2i::RuntimeError::Code::EOK, error.GetCode());
 
-  prediction.At (5, error);
+  prediction.At (12, error);
```

Comment:
This is very suspicious 🤔

Reply:
TFLite was indexing the number of elements and not the number of bytes stored. This example was using 3 elements in the input tensor, that's why originally the nonexistentindex test was using 5. Now, the internal indexing is using number of bytes (as all the other backends do), so the test was failing, because the total of bytes indexed are 12. I increased the value to 12 to make the test valid with this change. You can see this commit 3cafd90d06d868e2a5f2eb753662fe6769e5f603 and this dbc5f20e62a8faa16aebc2cda60d25ebf2236880